### PR TITLE
feat: sdk api improvements

### LINKED ISSUE #693: Namada SDK - examples project + improvements
Todo:
- [x] create examples project - ideally outside of the interface monorepo
- [x] add missing functionality to the sdk
  - [x] wallet functions
  - [x] masp functions
  - [ ] ~~support db name for web version~~(requires changes to the namada_sdk)

### PR DESCRIPTION
**Pull request by mateuszjasiuk**
Resolves #693 

Added couple of masp/wallet functions, mostly for nodejs support

### CODE REVIEW

File: packages/sdk/README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Namada SDK package
 
+## Note
+SDK will change frequently and changes may not be backward compatible. Sorry for that!
+NodeJS support is "experimental", in the sense that it is not used by us in any of our projects(yet).
```
Comment (jurevans) [inline packages/sdk/README.md:7]: Spacing on `projects(yet)`

File: packages/sdk/src/masp.ts [hunk 2]
```diff
@@ -47,4 +47,26 @@ export class Masp {
...
+  }
+
+  /**
+   * Add payment address to SDK wallet
+   * @async
+   * @param xvk - Extended viewing key
+   * @param alias - Alias for the key
+   * @returns void
+   */
+  async add_default_payment_address(xvk: string, alias: string): Promise<void> {
```
Comment (jurevans) [inline packages/sdk/src/masp.ts:69]: Should this be `addDefaultPaymentAddress` to match the other functions, as we're in TS?

**Review by jurevans — APPROVED** [review]
LGTM! Just left a couple minor comments

File: packages/sdk/src/masp.ts (hunk 2 above)
Comment (mateuszjasiuk) [reply packages/sdk/src/masp.ts:69]: Yeah! :)

**Pull request merged**